project/SparkBuild.scala

```diff
@@ -720,6 +720,7 @@ object Unidoc {
     // Skip class names containing $ and some internal packages in Javadocs
     unidocAllSources in (JavaUnidoc, unidoc) := {
       ignoreUndocumentedPackages((unidocAllSources in (JavaUnidoc, unidoc)).value)
+        .map(_.filterNot(_.getCanonicalPath.contains("org/apache/hadoop/hive/ql/io/orc")))
```

Comment:
We can exclude the `org.apache.hadoop.hive.ql.io.orc` package from Javadoc in this way like the other exclusions.